hubblestack/files/hubblestack_nova/oval_scanner.py

```diff
@@ -68,6 +69,11 @@ def audit(data_list, tags, labels, debug=False, **kwargs):
                 logging.info('The oval CVE scanner does not currently support {0}'.format(distro_name.capitalize()))
                 return ret
             local_pkgs = __mods__['pkg.list_pkgs']()
+            if distro_name in ('debian'):
+                logging.info('Referencing source packages to binary packages. This could take awhile...'.format(distro_name.capitalize()))
```

Comment:
We don't seem to be formatting anything here.

Reply:
I've fixed this.